stable/mariadb/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: mariadb
-version: 5.3.1
+version: 5.3.2
```

Following the semver versioning, I would have increased the MINOR version as you are adding a new feature:

> Given a version number MAJOR.MINOR.PATCH, increment the:
>
> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards-compatible manner, and
> PATCH version when you make backwards-compatible bug fixes.

```suggestion
version: 5.4.0
```

---

sure, would be happy to :)
thanks for the semver reminder